runtime-modules/common/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = 'substrate-common-module'
-version = '1.0.0'
+version = '1.0.1'
```

Comment:
semantically you have added something to the api and not a bug-fix/patch so it make more sense to bump to `1.1.0`

Reply:
Agree. I'll fix it.